PlayList.java

```diff
@@ -125,6 +214,13 @@ public String titleOfShortestTrack() {
     public void sortedInPlace() {
         // Uses the selection sort algorithm,  
         // calling the minIndex method in each iteration.
-        //// replace this statement with your code
-    }
+        for(int i=0;i<size-1;i++)
+        {
+         Track temp = this.tracks[minIndex(i)];
+         this.tracks[minIndex(i)]=this.tracks[i];
+         this.tracks[i]=temp;
```

Comment:
`minIndex` is kinda expensive method, it better to store the resulted value then use it whenever you need instead of calculating it twice.
```suggestion
         int min = minIndex(i);
         Track temp = this.tracks[min];
         this.tracks[min]=this.tracks[i];
         this.tracks[i]=temp;
```

Reply:
Thank you Aviv! Ive learned a lot from your comments :)